odilia/src/main.rs

```diff
@@ -90,13 +90,21 @@ async fn sigterm_signal_watcher(
 
 use atspi::events::document::LoadCompleteEvent;
 
+#[tracing::instrument]
+async fn change_priority(
+	SpeechPriority(priority): SpeechPriority,
+	Speech(ssip): Speech,
+) -> Result<(), odilia_common::errors::OdiliaError> {
+	ssip.send(SSIPRequest::SetPriority(priority)).await?;
+	Ok(())
+}
+
 #[tracing::instrument]
 async fn speak(
```

Comment:
these should be in a command preprocessor, where those handlers get executed in paralell and where command outputs are processed, spoken and whatever other commands we intend to support